### PR TITLE
bpo-46586: Fix documentation links

### PR DESCRIPTION
**Pull request by meersuri**
Incorrectly pointed to built-in the property() function as mentioned in #46586

https://bugs.python.org/issue46586

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@meersuri

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -545,15 +545,15 @@ always available.
    | :const:`mant_dig`   | DBL_MANT_DIG   | float precision: the number of base-``radix``    |
    |                     |                | digits in the significand of a float             |
    +---------------------+----------------+--------------------------------------------------+
-   | :const:`max`        | DBL_MAX        | maximum representable positive finite float      |
+   | :const:`!max`       | DBL_MAX        | maximum representable positive finite float      |
```
Comment (JelleZijlstra) [inline Doc/library/sys.rst]: I think this is better done in a separate PR, so we can backport to the other bugfix branches.

File: Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix enum.property link to point to the enum page rather than the built-in property() page
```
Comment (JelleZijlstra) [inline Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst]: Not sure this needs a NEWS entry at all.

Comment (meersuri) [reply Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst]: Yeah sorry about that, I used blurb-it for the first time and by the time I was done, I saw that the skip-news label had been added. Should I just make another commit to remove this file?

File: Doc/library/sys.rst (hunk 1 above)
Comment (meersuri) [reply Doc/library/sys.rst]: I see, so I guess that will require opening another issue and then the corresponding PR? Should I revert this commit here then?

Comment (JelleZijlstra) [reply Doc/library/sys.rst]: Yes, revert the commit here. It doesn't need an issue; trivial docs fixes don't need an issue. You can just attach the second PR to the same issue.

File: Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst (hunk 2 above)
Comment (JelleZijlstra) [reply Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst]: Yes, please!

File: Doc/library/sys.rst (hunk 1 above)
Comment (JelleZijlstra) [reply Doc/library/sys.rst]: Also, based on @zware's comment on the bug, it's going to be better to change these to just put `max` in double backticks, since they aren't supposed to link anywhere.

Comment (zware) [reply Doc/library/sys.rst]: That comment was @merwok, not me :wink:.  I agree with him, though.

Comment (JelleZijlstra) [reply Doc/library/sys.rst]: Oops, sorry!

**Review by JelleZijlstra — APPROVED** [review]

**Review by zware — APPROVED** [review]

**Review by ethanfurman — APPROVED** [review]
